src/cargo/util/toml/mod.rs

```diff
@@ -515,6 +501,21 @@ pub fn to_real_manifest(
         );
     };
 
+    if let Some(deps) = me
```

Comment:
This might have been better to keep together as we will now have to update two places

Reply:
This isn't the only thing duplicated and my hope is that future refactors will allow us to merge most of the logic back together.